nix/pkgs/default.nix

```diff
@@ -91,6 +91,12 @@ let
     inherit git-rev;
   };
 
+  # Thorp is an S3 sync tool used for deployments
+  thorp =
```

Comment:
Some remarks on this one:

* Actually `sources` is being passed as argument on the very top so there is no need to import it again (just noticed this also applies to other places in this file..)
* Also I think it would be better to only pass what is actually required instead of providing all of `sources`.
* Could you change this to use `pkgs.callPackage` to inject dependencies instead of passing all of `pkgs` ? I'll comment on the `default.nix` of thorp below as well.